README.md

```diff
@@ -40,13 +41,32 @@ Starter code is already provided in `controllers/main.controllers.js` file.<br>
 ## How to Contribute
 - Create the issue.
 - Fork the Repo.
+- Create a new Branch [git checkout -b "new branch name"]
 - Fix the issue.
 - Generate a Pull Request.
 - Wait for your PR to get merged.
 
+
+## Documentation
+Follow this Blog to know more about API Usage:
+[Documentation](wrike.com/project-management-guide/faq/why-should-i-use-api-in-project-management-software/)
+
+
+
 # Technologies to be used:
 - Nodejs
 
 ## Author & maintainer
 
 [Vighnesh Manjrekar](https://github.com/VighneshManjrekar)
+
+## 🚀 About Me
```

Comment:
GDSC-DMCE

Reply:
Please go through my changes

Reply:
Only update GDSC-DMCE keep rest of the part as it is